### ISSUE TITLE
There is no apparent way to know which image is groundtruth and which image has reflections

### ISSUE THREAD

**Issue by ashesh-0** [zhuyr97/Reflection_RemoVal_CVPR2024#7]
I tried using from the huggingface. After a bit of inspection, it was clear that individual labels in the training data correspond to one scene. however, it is not clear which image is the transmission image. Also, the training script provided here uses text files for selecting GT and input images with reflection. Those text files are missing from the huggingface. 

Thanks,

**Comment by MohAmin22** [comment]
@ashesh-0 Did you find those files?

**Comment by ashesh-0** [comment]
> [@ashesh-0](https://github.com/ashesh-0) Did you find those files?

No.
